### DISCUSSION TITLE
when they add "components"

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#5747] [Q&A] [answered]
**Hello!**

A few weeks ago the "components" was available but they removed it. 

And I wanted to know if they plan to remove it

**Comment by kyranet** [comment, marked as answer]
What do you mean removed it? Removed from where?

As for discord.js support, components will be supported with the v13 release, which will come from mid-end of June btw.

**Reply by ghost** [reply]
a ok ok :D
